client/app/main/student/partial/outreaches.html

```diff
@@ -51,8 +49,13 @@
         </dd>
         <dd ng-repeat="note in outreach.notes | orderBy:$index:true">
           <div class="input-group">
-            <input type="text" class="form-control" readonly value="{{note.note}}"/>
-            <span class="input-group-addon">{{note.date | date:'MM/dd/yy'}}</span>
+            <input type="text" class="form-control truncate" readonly value="{{note.note}}"/>
+            <span class="input-group-addon">{{note.date | date:'yyyy-MM-dd'}}</span>
```

Comment:
Let's change this date back to 'MM/dd/yy'

This matches 'shortDate' format for uib.